src/JuMP.jl

```diff
@@ -23,8 +23,6 @@ using Compat
 
 import Compat: UTF8String, view
 
-@compat import Base.show
-
```

Comment:
I thought this was still needed for the `writemime` patch? See [here](https://github.com/JuliaLang/Compat.jl/pull/219).